pyveda/vedaset/store/vedabase.py

```diff
@@ -89,17 +89,12 @@ def clean(self, count=None, include_background_tiles=True):
         """
         classes = self._vset.classes
         mltype = self._vset.mltype
-        Labelizer(self, mltype, count, classes, include_background_tiles).clean()
+        vv.labelizer.Labelizer(self, mltype, count, classes, include_background_tiles).clean()
 
     def preview(self, count=10, include_background_tiles=True):
-        """
-        Page through VedaCollection data and flag bad data.
-        Params:
-            count: the number of tiles to clean
-        """
```

Comment:
Why did you remove the docstring?

Reply:
it's technically not correct, and I just happened to notice that while I was working on this PR. I'll go in and fix the docstrings in dev directly, as there are quite a few that are wrong

Reply:
Ticket MLT-290 addresses this docstring work